src/LodeRunner.API.Test/IntegrationTests/Extensions/HttpClientExtension.cs

```diff
@@ -183,6 +184,51 @@ public static async Task<HttpResponseMessage> PostTestRun(this HttpClient httpCl
             return httpResponse;
         }
 
+        /// <summary>
+        /// Post an invalid TestRun.
+        /// </summary>
+        /// <param name="httpClient">The httpClient.</param>
+        /// <param name="postTestRunsUri">The post TestRun Uri.</param>
+        /// <param name="output">The output.</param>
+        /// <returns>the task.</returns>
+        public static async Task<HttpResponseMessage> PostInvalidTestRun(this HttpClient httpClient, string postTestRunsUri, ITestOutputHelper output)

```

Comment:
Seems like PostInvalidTestRun and PostTestRun can be refactored since the only main diference is the paylod, even you could use the generic PostEntity method that actually takes a payload as parameter

Reply:
Removed PostInvalidTestRun and re-used existing function.